docs/extension-system/_index.md

```diff
@@ -134,17 +134,22 @@ You can find predefined extension point ids in the extension points section belo
 #### Extension Points
 
 There are standardized components and places where extensions are being used automatically. The following are the ones that are currently provided
-by the ownCloud Web runtime or the `files` app.
+by the ownCloud Web runtime or the `files` app. If you decide to develop an extension which fulfills the type and allows the extensionPointId of the respective extension point, 
+your extension will be used automatically. 
 
 1. Left Sidebar for Navigation
-2. Right Sidebar in any file(s) context 
-3. Folder Views in the files app 
-4. Right click context menu in the files app 
-5. Batch actions in the files app
-6. Upload menu in the files app 
-7. Quick actions in the files list of the files app
-8. Search results in the search app
-9. Global progress bar for the global loading state. Extension point id `app.runtime.global-progress-bar`. Allows to render a single custom component.
+2. Any file(s) context (files app, viewer apps, editor apps)
+   1. Right sidebar. ExtensionPointId `app.files.sidebar`. Mounts extensions of type `sidebarPanel`. 
```

Comment:
I might be mixing things up, but shouldn't this be part of the next point "Files app"? Since the files app introduces this extension point?

Reply:
I wanted to express where the extension point is available (not so much by which application it's registered)... does that make sense? Can also be in the next section, I'm indifferent.

Reply:
then I'd say it should be a files.sidebar - not app.files.sidebar. I think app.files is misleading

Reply:
Yeah, happy you chose `global.` and implemented that already